common/scrape_utils.py

```diff
@@ -225,3 +225,25 @@ def check_missing_pdfs(meeting_links, dir=None):
     extra_pdfs = downloaded_pdfs - expected_pdfs
 
     return missing_links, extra_pdfs
+
+
+def download_pdf(year, date, url, dir=None):
+    if not dir:
+        dir = Path.cwd() / "pdf_files"
+
+    year_dir = dir / year
+    year_dir.mkdir(parents=True, exist_ok=True)
+
+    pdf_name = date.replace("-", "_") + ".pdf"
+    pdf_file = year_dir / pdf_name
+    response = requests.get(url)
```

Comment:
Instead of using the try/except block when writing the data, we could use `response.raise_for_status()` which only raises an exception if we get an error (e.g., 4XX or 5XX response codes). This also raises an error with the specific http error code instead of the `TypeError` which doesn't tell you _why_ the request failed, just that it did.

Reply:
Good suggestion! I wound up incorporating `response.raise_exceptions()` in the try/except block which starts on line 235 in my latest commit. What's weird is that the file you commented didn't have that try/except block in it even though it was part of the code when I submitted the PR. Should be in there now though